docs/docs/product-documentation.md

```diff
@@ -75,53 +71,48 @@ The strategic aim of the Swapchain team is to develop an HTLC-backed cryptocurre
 
 #### 2.2.1. User personas
 
 !(Description of user persona pathway)[static/img/UserPersona.jpg]
 Figure 1: User Persona Pathway Diagram (Swapchain, 2020a) [3]
 
-A typical user persona for the Swapchain would be Bob/Alice. Bob/Alice, 27 years old, just recently graduated from his/her (technological) studies at a university with a degree. Two months ago, he/she entered professional life in a corporation where he/she is in an occupation situated at the interface between tech and business, such as technical procurement or process optimisation. As he/she just recently began to earn a regular wage summing up to 47.000 € gross salary annually, he/she is now looking for investment opportunities - and as a tech-savvy person also interested in stocks, exchange rate gains and comparable investments, cryptocurrencies have (besides the technology itself) sparked his/her interest. Nonetheless, he/she is self-explanatory a responsible person that does not want to invest his/her first few wages in sketchy investment options risking to lose it all. Therefore, a sufficient level of security is inevitable to persuade Bob/Alice to invest in these assets and the underlying ACCS technology. Moreover, Bob/Alice is concerned with data protection and thus convinced that even when it comes to web-based transactions, as much information as possible should remain in his/her area of influence without being forwarded to third parties without prior consent.
+A typical user persona for the Swapchain would be Bob/Alice. Bob/Alice, 27 years old, just recently graduated from his/her (technological) studies at a university with a degree. Two months ago, he/she entered professional life in a corporation where he/she is in an occupation situated at the interface between tech and business, such as technical procurement or process optimisation. As he/she just recently began to earn a regular wage summing up to 47.000 € gross salary annually, he/she is now looking for investment opportunities - and as a tech-savvy person also interested in stocks, exchange rate gains and comparable investments, cryptocurrencies have (besides the technology itself) sparked his/her interest. Nonetheless, he/she is self-explanatory a responsible person that does not want to invest his/her first few wages in sketchy investment options risking to lose it all. Therefore, a sufficient level of security is inevitable to persuade Bob/Alice to invest in these assets and the underlying ACCS technology. Moreover, Bob/Alice is concerned with data protection and thus convinced that even when it comes to web-based transactions, as much information as possible should remain in his/her area of influence without being forwarded and stored by third parties.
```

Comment:
I think this paragraph might be too detailed. I would rather say that Alice/Bob has been trading with crypto for quite a while and is well-educated with regards to this topic. I mean we do not expect our users to possess a college degree in order to use swapchain, but a decent knowledge regarding the respective technology, right?

Reply:
That's a good point. I would write it the same way as Mads:

Alice/Bob has been trading with crypto for quite a while and is well-educated with regards to this topic.